packages/eas-cli/src/commands/channel/rollout.ts

```diff
@@ -105,7 +105,18 @@ async function startRolloutAsync({
       validate: value => (value ? true : validationMessage),
     }));
   }
-  const branch = await getBranchByNameAsync({ appId: projectId, name: branchName! });
+
+  const {
+    app: {
+      byId: { updateBranchByName: branch },
+    },
+  } = await BranchQuery.getBranchByNameAsync({ appId: projectId, name: branchName! });
```

Comment:
Please refactor the code so it doesn't use `!`. This often leads to errors / hard to debug issues. I took a look at the code and it should be really easy to refactor.

Reply:
Yea, great point.

I looked around a bit and decided to follow this pattern: https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/commands/build/configure.ts#L42